src/ngx_http_lua_args.c

```diff
@@ -19,6 +19,56 @@ static int ngx_http_lua_ngx_req_set_uri_args(lua_State *L);
 static int ngx_http_lua_ngx_req_get_post_args(lua_State *L);
 
 
+static ngx_inline ngx_uint_t
```

Comment:
Please note that in nginx's code base, it's usually not using `ngx_uint_t` for boolean types. Maybe you should just use `ngx_int_t` and then returning `NGX_OK` and `NGX_ERROR` instead.